pkg/provisioner/templates/kubernetes.go

```diff
@@ -168,6 +162,45 @@ echo "you can now access the cluster with:"
 echo "ssh -i <your-private-key> ubuntu@${INSTANCE_ENDPOINT_HOST}"
 `
 
+const kubeadmTemplate = `apiVersion: kubeadm.k8s.io/v1beta4
```

Comment:
Why not *only* require a local config file? Is supporting both a config file and a templated config is worth it?

Reply:
Does this example shows why we should *NOT* support a config file -- or at least not at this stage.

Since there are some options that we MUST set as part of the tooling, how do we handle collisions when a user is configuring them?

Maybe we should focus on the purpose of holodeck: from my perspective, it is supposed to set up and configure *infrastructure* for the components we're testing. It is *not* a general purpose tool for deploying and configuring k8s. Is this accurate? In that case, does adding support for a user-specified kubernetes config serve this purpose?

Reply:
I agree with you; I will remove the config file option. The main reason we need this is to enable feature-flags on our k8s test environments

Reply:
I have removed the option to provide a kubeadmin config file. PTAL